helpers/debounce.js

```diff
@@ -0,0 +1,16 @@
+const debounce = (func, timeout) => {
+  let timer;
+
+  return () => {
+    clearTimeout(timer);
+    timer = setTimeout(() => {
+      // if (!timer) {
```

Comment:
We can remove the comments if not needed

Reply:
Resolved